docs/_config.yml

```diff
@@ -0,0 +1,25 @@
+#theme: "just-the-docs"
+remote_theme: pmarsceill/just-the-docs
+exclude: ["node_modules/", "*.gemspec", "*.gem", "Gemfile","README.md", "Gemfile.lock", "package.json", "package-lock.json",  "script/", "LICENSE.txt", "lib/", "bin/", "README.md", "Rakefile"]
+
+
+aux_links:
+  "VulnerableApp":
+    - "https://github.com/SasanLabs/VulnerableApp"
+
+title: Vulnerable App
+description: Documentation for VulnerableApp
+baseurl: "/VulnerableApp/" # the subpath of your site, e.g. /blog
+url: "https://hemantgs.github.io/" # the base hostname & protocol for your site, e.g. http://example.com"
```

Comment:
i am not sure if this required ?

Reply:
that was needed to host it in my repo , I have changed that now